File: app/src/main/java/net/imatruck/dsmanager/models/DSStatsInfoData.java
package net.imatruck.dsmanager.models;

import com.google.gson.annotations.SerializedName;


@SuppressWarnings("unused")
public class DSStatsInfoData {

    @SerializedName("speed_download")
    private double speedDownload;

    @SerializedName("speed_upload")
    private double speedUpload;

    public double getSpeedDownload() {
        return speedDownload;
    }

    public double getSpeedUpload() {
        return speedUpload;
    }
}

File: app/src/main/java/net/imatruck/dsmanager/models/DSStatsInfoData.kt
package net.imatruck.dsmanager.models

import com.google.gson.annotations.SerializedName


class DSStatsInfoData {
    @SerializedName("speed_download")
    val speedDownload: Double = 0.toDouble()
    @SerializedName("speed_upload")
    val speedUpload: Double = 0.toDouble()
}
